GitUI/CommandsDialogs/FormDeleteRemoteBranch.cs

```diff
@@ -63,13 +63,11 @@ private void OkClick(object sender, EventArgs e)
                     }
                 }
 
-                foreach (var remoteGroup in selectedBranches.GroupBy(b => b.Remote))
+                foreach (var (remote, branches) in selectedBranches.GroupBy(b => b.Remote))
                 {
-                    string remote = remoteGroup.Key;
-
                     EnsurePageant(remote);
 
-                    var cmd = new GitDeleteRemoteBranchesCmd(remote, remoteGroup);
+                    var cmd = new GitDeleteRemoteBranchesCmd(remote, branches);
```

Comment:
This looks a bit different to the other cases where `.Key` (etc) was removed. That's because when destructuring `IGrouping<TKey,TElement>` the second value is the grouping itself, re-cast as `IEnumerable<TElement>`.